Assets/MixedRealityToolkit/Inspectors/ControllerPopupWindow.cs

```diff
@@ -218,12 +219,12 @@ public static void Show(MixedRealityControllerMapping controllerMapping, Seriali
                     }
                 };
 
-                File.WriteAllText($"{Application.dataPath}{EditorWindowOptionsPath}", JsonUtility.ToJson(empty));
```

Comment:
Output functions **should not change**. keep the app data path as these are unity constants. especially as this pr is about fixing reading of files.

Reply:
I'm not sure that we're on the same page here about what this is doing.

Today, if you have MRTK in the root Assets folder, this thing works because Application.dataPath is just the assets folder. Note that, this is **editor specific code that runs in editor**

This obviously doesn't work if it's just in a subfolder of Assets. This is what this fixes. It happens to be in this case that the input == a json file about the controller window, and the output == a json file about the controller window, so they are literally the same thing.

Reply:
Another way of saying this is that, I think that the comment here being left is a generally valid statement but it doesn't apply here - if you have an alternative suggestion that leaves this the same (which would, FWIW, be broken), prefer leaving concrete suggestions instead of just generalisms.